osu.Game.Rulesets.Osu/Skinning/OsuLegacySkinTransformer.cs

```diff
@@ -132,6 +132,12 @@ public Drawable GetDrawableComponent(ISkinComponent component)
                                 return SkinUtils.As<TValue>(new BindableFloat(LEGACY_CIRCLE_RADIUS));
 
                             break;
+
+                        case OsuSkinConfiguration.HitCircleOverlayAboveNumber:
+                            // Quote from https://osu.ppy.sh/help/wiki/Skinning/skin.ini#%5Bgeneral%5D
+                            // Old command: HitCircleOverlayAboveNumer (with typo) still works for legacy support
+                            var rv = source.GetConfig<OsuSkinConfiguration, TValue>(OsuSkinConfiguration.HitCircleOverlayAboveNumber);
+                            return rv ?? source.GetConfig<OsuSkinConfiguration, TValue>(OsuSkinConfiguration.HitCircleOverlayAboveNumer);
```

Comment:
```suggestion
                            return source.GetConfig<OsuSkinConfiguration, TValue>(OsuSkinConfiguration.HitCircleOverlayAboveNumber) ?? source.GetConfig<OsuSkinConfiguration, TValue>(OsuSkinConfiguration.HitCircleOverlayAboveNumer);
                            
```

@mcendu You mean like this?

Reply:
I'd say that's better (also gets rid of the very cryptic `rv` variable name).